lib/streamlit/runtime/context.py

```diff
@@ -205,3 +205,20 @@ def cookies(self) -> StreamlitCookies:
 
         cookies = session_client_request.cookies
         return StreamlitCookies.from_tornado_cookies(cookies)
+
+    @property
+    def timezone(self) -> str:
+        ctx = get_script_run_ctx()
+        if ctx is None:
+            return None
+
+        session_client = runtime.get_instance().get_client(ctx.session_id)
+
+        if session_client is None:
+            return None
+
+        return (
+            runtime.get_instance()
+            ._session_mgr.get_session_info(ctx.session_id)
+            .session._client_state.context_info.timezone
+        )
```

Comment:
I think we can put the timezone info directly in `ScriptRunContext` (similar to query_string) so that we don't have to extract it from client state

Reply:
Done